go/pkg/chunker/chunker.go

```diff
@@ -134,6 +134,7 @@ func (c *Chunker) FullData() ([]byte, error) {
 	}
 	// Cache contents so that the next call to FullData() doesn't result in file read.
 	c.contents, err = ioutil.ReadAll(c.r)
+	c.r.Close()
```

Comment:
Should this be `defer c.r.Close()` at the beginning of this function?

Reply:
I don't think so. I'm only closing when I do a full read (and reach EOF).